src/django/api/models/reference_image.py

```diff
@@ -9,7 +9,7 @@
 class ReferenceImage(models.Model):
     filename = models.CharField(max_length=255, blank=True)
     boundary = models.ForeignKey(Boundary, on_delete=models.PROTECT)
-    uploaded_at = models.DateTimeField()
+    uploaded_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Now that we intend to save to the backend on every change, I think adding a field for "order added" might be redundant. I'm thinking we can just use uploaded_at (or for race conditions, I suppose id is safer).

The "order added" on the frontend will be subordinate to the order on the server anyway, since multiple submissions for the same boundary can be worked on simultaneously by different users and be updating the backend contemporaneously.

Reply:
Makes sense!